src/Entity/PartyEvent.php

```diff
@@ -20,34 +25,37 @@
         new Post(),
         new Put(),
         new Delete(),
-    ]
+    ],
+    formats: ['json'],
+    collectDenormalizationErrors: true
 )]
+#[ApiFilter(filterClass: SearchFilter::class, properties: ['name' => 'partial'])]
+#[ApiFilter(RangeFilter::class, properties: ['date'])]
+#[ApiFilter(OrderFilter::class, properties: ['date' => 'ASC', 'name' => 'ASC'])]
 #[ORM\Entity(repositoryClass: PartyEventRepository::class)]
 class PartyEvent
 {
 
     #[ORM\Id]
     #[ORM\Column(length: 255, unique: true)]
+    #[Assert\NotBlank]
     private string $uid;
 
     #[ORM\Column(length: 255)]
     #[Assert\NotBlank]
     private string $name;
 
-    #[ORM\Column(type: Types::DATETIME_MUTABLE)]
-    private ?\DateTimeInterface $date = null;
-
+    #[ORM\Column(type: Types::BIGINT, nullable: true)]
+    private ?int $date = null;
```

Comment:
### ❗ Issue: It can't be nullable

I will send u `date` in 100% cases... in other case - just create it by yourself :)